portfolio/src/main/webapp/style.css

```diff
@@ -1,29 +1,50 @@
+@import url("https://fonts.googleapis.com/css?family=DM+Sans:400,500|Jost:400,500,600&display=swap");
```

Comment:
Same for this file, let's follow https://google.github.io/styleguide/htmlcssguide.html#Indentation for the indentations.